qiskit/circuit/quantumcircuit.py

```diff
@@ -294,27 +298,6 @@ def data(self) -> QuantumCircuitData:
         """
         return QuantumCircuitData(self)
 
-    @property
-    def op_start_times(self) -> List[int]:
-        """Return a list of operation start times.
-
-        This attribute is enabled once one of scheduling analysis passes
-        runs on the quantum circuit.
-
-        Returns:
-            List of integers representing instruction start times.
-            The index corresponds to the index of instruction in :attr:`QuantumCircuit.data`.
-
-        Raises:
-            AttributeError: When circuit is not scheduled.
-        """
-        if self._op_start_times is None:
-            raise AttributeError(
-                "This circuit is not scheduled. "
-                "To schedule it run the circuit through one of the transpiler scheduling passes."
-            )
-        return self._op_start_times
-
```

Comment:
Please can this be restored to the correct place, so it's clear in the history that this PR doesn't change this code.

Reply:
From what I understand the mypy expects setter and getter follow each other, which also makes sense for me

Reply:
Indeed, https://github.com/python/mypy/issues/1465